src/seed/bootstrap.ts

```diff
@@ -87,7 +87,7 @@ async function bootstrapDatabases() {
 
     if (!(await songThresholdReached(db))) {
         logger.info(`Downloading minimum threshold (${SONG_DOWNLOAD_THRESHOLD}) songs`);
-        await downloadAndConvertSongs(SONG_DOWNLOAD_THRESHOLD);
+        await downloadAndConvertSongs(DEFAULT_SONGS_PER_ARTIST, SONG_DOWNLOAD_THRESHOLD);
```

Comment:
`SONG_DOWNLOAD_THRESHOLD` is is specific to the bootstrap sequence, as should `DEFAULT_SONGS_PER_ARTIST`. Also name it so that it's more obvious it's b-sides.